site/content/en/docs/tutorials/setup_minikube_in_china.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "setup minikube in China"
```

Comment:
make the title "Setting up minikube in China"

Reply:
sure

Reply:
Updated.